packages/yoastseo/spec/languageProcessing/helpers/word/getWordsForHTMLParserSpec.js

```diff
@@ -68,6 +68,11 @@ const testCases = [
 		text: "[caption id=\"attachment_3341501\" align=\"alignnone\" width=\"300\"]",
 		expectedResult: [ "[", "caption", " ", "id=\"attachment_3341501", "\"", " ", "align=\"alignnone", "\"", " ", "width=\"300", "\"", "]" ],
 	},
+	{
+		description: "correctly tokenizes a shortcode with text",
+		text: "[caption]test[/caption]",
+		expectedResult: [ "[", "caption", "]", "test", "[", "/", "caption", "]" ],
```

Comment:
If slash `/` is removed from the word boundaries regex, this unit test will also need to be adjusted 👇🏽 
```suggestion
		expectedResult: [ "[", "caption", "]", "test", "[", "/caption", "]" ],
```